src/main/java/csci432/ItemTracker.java

```diff
@@ -0,0 +1,84 @@
+package csci432;
+import java.util.concurrent.*;
+import java.util.*;
+import java.awt.image.BufferedImage;
+
+/**
+ * Class used for identifying moving objects, called blobs, based off
+ * of images that have been passed through Sigma-Delta and edge
+ * detection algorithms.
+ *
+ **/
+public class ItemTracker<T> {
+    Queue<BufferedImage> queue;
+    List<Blob> currentBlobs;
+
+    public ItemTracker() {
+	queue = new LinkedBlockingQueue<BufferedImage>();
+	currentBlobs = new LinkedList<Blob>();
+    }
+
+    /**
+     * The method that does stuff
+     **/
+    public void run() {
+	while(!queue.isEmpty()) {
```

Comment:
I think I lied here, super super sorry. I think what you originally had commented out was actually correct...